conf/messages

```diff
@@ -55,6 +55,24 @@ checkingRecords.ffid=Identifying file formats
 fileChecksResults.header=Record check results
 fileChecksResults.title=Record check results
 fileChecksResults.description=Here are your results
+transferSummary.header=Transfer Summary
+transferSummary.title=Transfer Summary
+transferSummary.seriesReference=Series reference
+transferSummary.consignmentReference=Consignment reference
+transferSummary.transferringBody=Transferring body
+transferSummary.printableReport=Printable Report
+transferSummary.download=Download
+transferSummary.filesUploadedForTransfer=Files uploaded for transfer
+transferSummary.openRecords=I confirm that in this version of TDR, all records that I transfer are open
```

Comment:
Is this wording correct: "I can confirm **in this version of TDR**, all records that I transfer are open"

Reply:
Yes, this is the text from the updated [wireframe](https://h81xos.axshare.com/reviewandconfirm.html)